redcap/methods/base.py

```diff
@@ -353,7 +353,6 @@ def _return_data(
         ],
         format_type: Literal["json", "csv", "xml", "df"],
         df_kwargs: Optional[Dict[str, Any]] = None,
-        record_type: Literal["flat", "eav"] = "flat",
```

Comment:
Is there a reason you've removed this argument? I would still like to be able to have the user return a data frame for the `eav` record type option

Reply:
Not sure anymore. I guess, I did not see, that `content = 'report` when calling `_load_data()` from `export_records()`. I modified the code to leave the `df_kwargs` unchanged for the `eav` record type option.